scripts/copper/slab/prepare.py

```diff
@@ -83,12 +89,47 @@ def _prepare_free_layer_convergence(atom: Atoms, data_path: Path) -> None:
     maybe_copy_files_to_hpc(data_path, PosixPath(data_path.as_posix()))
 
 
+def _prepare_slab_width_convergence(
+    atom: Atoms,
+    data_path: Path,
+) -> None:
+    calculators = list[Castep]()
+    prepare_clean_directory(data_path)
+
+    slab = get_surface(
+        atom,
+        (1, 1, 1),
+        n_fixed_layer=4,
+        n_free_layer=4,
+        n_vacuum_layer=5,
+    )
+
+    slab_position = get_top_position(slab)
+    rel_position = [0, 0, 1.0]
+    na_atom = prepare_na_atom(slab_position, rel_position)
```

Comment:
comments yesterday about vaccum height, and atom needs to be on the side of surface which is free